### DISCUSSION TITLE
Monitoring Interfaces Config

### DISCUSSION THREAD

**Discussion by fogel-j** [Security-Onion-Solutions/securityonion#12491] [General] [answered]
I know that `so-monitor-add` can be used to add a monitoring interface. During initial imaging we have the ability to add monitoring interfaces. These monitoring interfaces are routed to `bond0`. How can we check what interfaces have been added? Is there an ability to delete a monitoring interface? I couldn't find this in the documentation.

**Comment by TotieBash** [comment, marked as answer]
I'm not in front of my system right now but I think "nmtui" should give you the answer.

**Reply by fogel-j** [reply]
Okay thanks! I now have a better understanding. `bond0` is configured as a 'Bond' interface that has 'Slave' interfaces which are the monitoring interfaces you set. For more info, this link was helpful - https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/networking_guide/sec-configure_bonding_using_the_text_user_interface_nmtui
